main.go

```diff
@@ -27,19 +27,19 @@ import (
 func main() {
 	veleroplugin.NewServer().
 		BindFlags(pflag.CommandLine).
-		RegisterBackupItemAction("velero.io/csi-snapshotter", newCSISnapshotter).
-		RegisterBackupItemAction("velero.io/volumesnapshot-backupper", newVolumeSnapshotBackupItemAction).
-		RegisterBackupItemAction("velero.io/volumesnapshotclass-backupper", newVolumesnapshotClassBackupItemAction).
-		RegisterBackupItemAction("velero.io/volumesnapshotcontent-backupper", newVolumeSnapContentBackupItemAction).
-		RegisterRestoreItemAction("velero.io/csi-restorer", newCSIRestorer).
-		RegisterRestoreItemAction("velero.io/volumesnapshotclass-restorer", newVolSnapClassRestorer).
-		RegisterRestoreItemAction("velero.io/volumesnapshotcontent-restorer", newVSCRestorer).
-		RegisterRestoreItemAction("velero.io/volumesnapshot-restorer", newVSRestorer).
+		RegisterBackupItemAction("velero.io/csi-pvc-backupper", newPVCBackupItemAction).
+		RegisterBackupItemAction("velero.io/csi-volumesnapshot-backupper", newVolumeSnapshotBackupItemAction).
+		RegisterBackupItemAction("velero.io/csi-volumesnapshotclass-backupper", newVolumesnapshotClassBackupItemAction).
+		RegisterBackupItemAction("velero.io/csi-volumesnapshotcontent-backupper", newVolumeSnapContentBackupItemAction).
+		RegisterRestoreItemAction("velero.io/csi-pvc-restorer", newPVCRestoreItemAction).
+		RegisterRestoreItemAction("velero.io/csi-volumesnapshotclass-restorer", newVolSnapClassRestorer).
+		RegisterRestoreItemAction("velero.io/csi-volumesnapshotcontent-restorer", newVSCRestorer).
+		RegisterRestoreItemAction("velero.io/csi-volumesnapshot-restorer", newVolumeSnapshotRestoreItemAction).
```

Comment:
minor nit but it'd be nice to move L37 up to just above L35, so the plugins are ordered the same for backup & restore.